include/ccf/ds/attestation_types.h

```diff
@@ -2,14 +2,26 @@
 // Licensed under the Apache 2.0 License.
 #pragma once
 
-#ifdef GET_QUOTE
+#if defined(INSIDE_ENCLAVE) && !defined(VIRTUAL_ENCLAVE)
 #  include <openenclave/attestation/attester.h>
 #  include <openenclave/attestation/custom_claims.h>
 #  include <openenclave/attestation/sgx/evidence.h>
 #  include <openenclave/attestation/verifier.h>
 
+#endif
+
+#include <array>
+
 namespace ccf
 {
+  static constexpr size_t attestation_report_data_size = 32;
```

Comment:
I don't think we want to assume these are necessarily the same across all platforms, they could vary, and we may want to take advantage of that.